pkg/events/event_ringbuffer.go

```diff
@@ -116,9 +116,20 @@ func min(a, b uint64) uint64 {
 	return m
 }
 
-// GetLastEventID returns the value of the unique id counter
+// GetLastEventID returns the value of the unique id counter.
+// If the buffer is empty, it returns 0.
 func (e *eventRingBuffer) GetLastEventID() uint64 {
-	return e.id
+	e.RLock()
+	defer e.RUnlock()
+	return e.getLastEventID()
+}
+
+// unlocked version of GetLastEventID()
+func (e *eventRingBuffer) getLastEventID() uint64 {
+	if e.id == 0 {
+		return 0
```

Comment:
Special case: no queues, no apps, no nodes, no nothing. It will never happen in real life (we can't return -1 due to "id" being uint64).